dist/tribute.js

```diff
@@ -422,6 +423,42 @@ var Tribute = function () {
                 throw new Error('No active state. Please use append instead and pass an index.');
             }
         }
+    }, {
+        key: "detach",
+        value: function detach(el) {
+            if (!el) {
+                throw new Error('[Tribute] Must pass in a DOM node or NodeList.');
+            }
+
+            // Check if it is a jQuery collection
+            if (typeof jQuery !== 'undefined' && el instanceof jQuery) {
+                el = el.get();
+            }
+
+            // Is el an Array/Array-like object?
+            if (el.constructor === NodeList || el.constructor === HTMLCollection || el.constructor === Array) {
+                var length = el.length;
+                for (var i = 0; i < length; ++i) {
+                    this._detach(el[i]);
+                }
+            } else {
+                this._detach(el);
+            }
+        }
+    }, {
+        key: "_detach",
+        value: function _detach(el) {
+            var _this3 = this;
+
+            this.events.unbind(el);
+            this.menuEvents.unbind(el.tributeMenu);
+
+            setTimeout(function () {
+                el.removeAttribute('data-tribute');
```

Comment:
Thanks for all the hard work on this feature.

I'm getting a warning from the `attach` function when I use the `detach` and `attach` function back to back:
```
tribute.detach(element);
tribute = new Tribute({
    collection: newValue
});
tribute.attach(element);
```
`Tribute was already bound to DIV`

I suspect it's the setTimeout function doing this, but was there a reason for it?